binary_transparency/firmware/cmd/ft_monitor/impl/ft_monitor.go

```diff
@@ -65,15 +71,23 @@ func Main(ctx context.Context, opts MonitorOpts) error {
 
 	c := client.ReadonlyClient{LogURL: ftURL}
 
-	// TODO(mhutchinson): This checkpoint and tracker for number of processed entries
-	// should be serialized so the monitor persists its golden state between runs.
-	// In particular this will prevent the annotator from adding multiple annotations
-	// for the same firmware each time the monitor is started.
+	// Initialize the checkpoint from persisted state.
 	var latestCP api.LogCheckpoint
-	var head uint64
+	if state, err := ioutil.ReadFile(opts.StateFile); err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("failed to read state: %w", err)
+		}
+		// Silently continue if the file doesn't exist.
```

Comment:
Perhaps we should generate a Warning here and then progress?

Reply:
Good call; done.